app/models/like.rb

```diff
@@ -1,4 +1,7 @@
 class Like < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  belongs_to :user
   belongs_to :post
+  def increment_likes_count
+    post.increment!(:likes_counter)
+  end
```

Comment:
[OPTIONAL] Nice work with this method 👍 It would be even better if you made use of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) to call this method, specifically the `after_save` callback. What it does is call the method whenever a new post is saved to the database automatically.
To use it, you can use the following line:
```
after_save increment_likes_count 
```
It's also best practice to make the method in question a private method.